contrib/pg_bm25/Dockerfile

```
@@ -0,0 +1,29 @@
+ARG PG_VERSION=15
+FROM quay.io/coredb/pgrx-builder:pg${PG_VERSION}-pgrx0.9.7
+USER root
+
+ARG RELEASE=v0.2.21
+
+# Extension build dependencies
+RUN apt-get update && apt-get install -y \
+	build-essential
+
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+# Set default Rust version
+RUN /root/.cargo/bin/rustup default stable && /root/.cargo/bin/rustup override set 1.72.0
```

```suggestion
RUN /root/.cargo/bin/rustup default stable
```

Note: this is a temporary hack, we need to rebuild the quay.io images so that the pgrx built in the image gets build by Rust 1.73